cleanup-passes/polymer-json.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as semver from 'semver';
+
+import {register} from '../cleanup-pass';
+import {ElementRepo} from '../element-repo';
+import {existsSync, makeCommit, writeToConfig} from './util';
+
+interface PolymerConfig {
```

Comment:
This type isn't right, lint is an object with, I think a `rules` property. Fortunately, you can just the type from polymer-project-config

The JSON format type is called something like `Options`. Maybe `ProjectOptions`

Can also use that library for validation with the static methods on ProjectConfig, though it'll just do a runtime type check, so that's probably not necessary.